src/projects/base/provider/stream.cpp

```diff
@@ -248,7 +250,21 @@ namespace pvd
 
 	bool Stream::AdjustRtpTimestamp(uint32_t track_id, int64_t timestamp, int64_t max_timestamp, int64_t &adjusted_timestamp)
 	{
+		auto stream = std::dynamic_pointer_cast<pvd::PullStream>(GetSharedPtr());
```

Comment:
This codes must be in the UNDER_DECISION block below. Otherwise this code will be called every time.